File: core/src/main/java/com/ustadmobile/core/controller/SelectLanguageDialogPresenter.java
package com.ustadmobile.core.controller;

import com.ustadmobile.core.impl.UmAccountManager;
import com.ustadmobile.core.db.UmAppDatabase;
import com.ustadmobile.core.impl.UstadMobileSystemImpl;

import java.util.Hashtable;

import com.ustadmobile.core.view.SelectLanguageDialogView;


/**
 * Presenter for SelectLanguageDialog view
 **/
public class SelectLanguageDialogPresenter extends UstadBaseController<SelectLanguageDialogView> {

    UmAppDatabase repository;


    public SelectLanguageDialogPresenter(Object context, Hashtable arguments, SelectLanguageDialogView view) {
        super(context, arguments, view);

        repository = UmAccountManager.getRepositoryForActiveAccount(context);


    }

    @Override
    public void onCreate(Hashtable savedState) {
        super.onCreate(savedState);
    }


}

File: core/src/commonMain/kotlin/com/ustadmobile/core/controller/SelectLanguageDialogPresenter.kt
package com.ustadmobile.core.controller

import com.ustadmobile.core.db.UmAppDatabase
import com.ustadmobile.core.impl.UmAccountManager
import com.ustadmobile.core.view.SelectLanguageDialogView


/**
 * Presenter for SelectLanguageDialog view
 */
class SelectLanguageDialogPresenter(context: Any,
                                    arguments: Map<String, String?>,
                                    view: SelectLanguageDialogView)
    : UstadBaseController<SelectLanguageDialogView>(context, arguments, view) {

    internal var repository: UmAppDatabase

    init {
        repository = UmAccountManager.getRepositoryForActiveAccount(context)

    }

    override fun onCreate(savedState: Map<String, String?>?) {
        super.onCreate(savedState)
    }


}
